LiteCore/Storage/KeyStore.hh

```diff
@@ -84,6 +85,10 @@ namespace litecore {
         /// Returns true if this is a valid collection name, or a scope plus a collection name.
         [[nodiscard]] static bool isValidCollectionNameWithScope(slice name);
 
+        [[nodiscard]] static std::string hexName(string_view name) {
```

Comment:
This function isn't specific to KeyStore; I would put it in SecureDigest.hh instead.

Reply:
will do